libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailFragment.java

```diff
@@ -334,10 +339,18 @@ protected void next(String email) {
         }
 
         if (isValidEmail(email)) {
+            mIsValidEmail = true;
             startProgress();
             mRequestedEmail = email;
             mDispatcher.dispatch(AccountActionBuilder.newIsAvailableEmailAction(email));
         } else {
+            mIsValidEmail = false;
+            showEmailError(R.string.email_invalid);
+        }
+    }
+
+    private void showErrorIfEmailInvalid() {
+        if (!mIsValidEmail && mEmailInput.getEditText().getText().length() > 0) {
```

Comment:
Hi @FernandaCG Did you check the length of the `EditText` so that the warning doesn't get shown when the activity is loaded? I tested this and that is the behavior I saw.  Good job 😄 We can take this a bit further though and utilize the `Boolean` object wrapper so that the initial value can be null before the email validation is done because if you click next and it shows the warning and you rotate the screen the warning disappears and we don't want that to happen 😄

Reply:
I'm not so sure how to do that. I try validating the `mEmailInput.getEditText().getText() == null `or setting the `mEmailInput` to `null`, but not works. 
I need some help 😄

Reply:
Oh I meant to just change the `private boolean mIsValidEmail = false;` to `private Boolean mIsValidEmail;`so now you can check if it's null before showing the error. Makes sense? Let me know.

Reply:
I understand, thank you for your help.
I just commit my changes